### ISSUE TITLE
Loss function used in yolo training

### ISSUE THREAD

**Issue by kavindujp** [pjreddie/darknet#2541]
I want to implement and train yolov3 using tensorflow. I want to know about loss function used in YOLO. But I could not find information regarding that. Anyone know what is the Loss function used in original YOLOV3 and what is the optimization algorithm used in YOLOV3 while training in darknet framework?
